util.php

```diff
@@ -3,6 +3,7 @@
 use ast\flags;
 
 const AST_DUMP_LINENOS = 1;
+const AST_DUMP_EXCLUDE_DOC = 2;
```

Comment:
I would prefer to call this `AST_DUMP_EXCLUDE_DOC_COMMENT`.

Reply:
Changed the constant name
5cbf249